RadixWallet/Clients/CloudBackupClient/CloudBackupClient+Live.swift

```diff
@@ -142,20 +143,55 @@ extension CloudBackupClient {
 			try? userDefaults.setLastCloudBackup(result, of: profile)
 		}
 
+		@Sendable
+		func performAutomaticBackup(_ profile: Profile, timeToCheckIfClaimed: Bool) async {
+			let needsBackUp = profile.appPreferences.security.isCloudProfileSyncEnabled && profile.header.isNonEmpty
+			let lastBackup = userDefaults.getLastCloudBackups[profile.id]
+			let lastBackupSucceeded = lastBackup?.result == .success && lastBackup?.saveHash == profile.saveHash
+
+			let shouldBackUp = needsBackUp && !lastBackupSucceeded
+			let shouldCheckClaim = shouldBackUp || timeToCheckIfClaimed
+
+			guard shouldBackUp || shouldCheckClaim else { return }
+
+			let existingRecord = try? await fetchProfileRecord(.init(recordName: profile.id.uuidString))
+
+			let backedUpID = { try? existingRecord.map(getProfileHeader)?.lastUsedOnDevice.id }
+
+			let didTransferBackProfile: Bool
+			if shouldCheckClaim, let id = backedUpID(), await !profileStore.isThisDevice(deviceID: id) {
+				let action = await overlayWindowClient.scheduleFullScreen(.init(root: .claimWallet(.init())))
+				didTransferBackProfile = action == .claimWallet(.didTransferBack)
+			} else {
+				didTransferBackProfile = false
+			}
+
+			guard shouldBackUp || didTransferBackProfile else { return }
+
+			await backupProfileAndSaveResult(existingRecord: existingRecord)
```

Comment:
Why isn't profile passed down to be backed up?

Reply:
It's left over from a previous flow, as it is now there is no reason to get the profile again. Will change it back.